app/controllers/campaignController.js

```diff
@@ -0,0 +1,55 @@
+const postMessageMethod = 'chat.postMessage'
+const listUsersMethod = 'users.list'
+const Slack = require('slack-node')
+
+router.post('/test-send', function (req, res) {
+  url = getUrlForRequest(configResolver.getConfigVariable('TEST_USER'));
+  makeHttpsGetRequest(url);
+	res.send('"How was your week?" - notifications sent to private channels. Thanks!');
+})
+
+router.post('/test-send-many', function (req, res) {
+  setCampaign();
+  sendToMany();
+	res.send('"How was your week?" - notifications sent to private channels. Thanks!');
+})
+
+function setCampaign() {
+  campaignId = 'C' + Date.now();
```

Comment:
Global variables - big `no-no` in node apps. 

1. If you have more then one instance of an app, it will stop working
2. If there is a new request, it will override it